tests/Rule/LatteTemplatesRule/PresenterWithoutModule/LatteTemplatesRuleForPresenterTest.php

```diff
@@ -137,6 +137,12 @@ public function testComponents(): void
                 23,
                 'default.latte',
             ],
+            [
+                'Component with name "noType" have no type specified.',
+                25,
+                'default.latte',
+                'Use phpstan/nette and define return type of createComponentNoType method.',
```

Comment:
Do we need phpstan-nette for this?

Reply:
We need return type extension from phpstan-nettethat knows how to detect getControl return type from createControl

Reply:
Or do you detect it separately?

Reply:
I just use phpstan function to get return type of method (I think it doesn't need phpstan-nette because we don’t have it installed in tests and it works)

https://github.com/efabrica-team/phpstan-latte/blob/main/src/Collector/ComponentCollector.php#L93

Reply:
Ok I thought you use phpstan-nette to get this info. But you detect it yourslef so it is not needed